proto/backup.proto

```diff
@@ -39,17 +39,25 @@ message BackupMeta {
 
 message File {
     string name = 1;
-    uint32 crc32 = 2;
+    uint64 content_checksum = 2;
```

Comment:
shoud we change it to `string`, if we use hash function like [sha-1](https://docs.rs/rust-crypto/0.2.36/crypto/sha1/index.html)

Reply:
prefer to use `bytes`

Reply:
Or maybe `bytes`?

Reply:
OK, I will use `bytes`